<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1128</width>
    <height>720</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #FEF8E8;
border-radius: 2px</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>131</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">opacity: 0.2;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../Desktop/Projeto Desktop/Taskzen/image/Logo.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="Taskzen">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat SemiBold</family>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #1F3043</string>
    </property>
    <property name="text">
     <string>T A S K Z E N</string>
    </property>
   </widget>
   <widget class="QLabel" name="Cadastro">
    <property name="geometry">
     <rect>
      <x>468</x>
      <y>130</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat ExtraBold</family>
      <pointsize>22</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <stylestrategy>PreferAntialias</stylestrategy>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #E33C2A</string>
    </property>
    <property name="text">
     <string>CADASTRO</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="txt_usuario">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>210</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Medium</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #1F3043;
</string>
    </property>
    <property name="text">
     <string>Usu√°rio:</string>
    </property>
   </widget>
   <widget class="QLabel" name="txt_email">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>310</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Medium</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #1F3043;
</string>
    </property>
    <property name="text">
     <string>Email:</string>
    </property>
   </widget>
   <widget class="QLabel" name="txt_password">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>400</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Medium</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #1F3043;
</string>
    </property>
    <property name="text">
     <string>Senha:</string>
    </property>
   </widget>
   <widget class="QLabel" name="txt_password2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>480</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Medium</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #1F3043;
</string>
    </property>
    <property name="text">
     <string>Confirmar Senha:</string>
    </property>
   </widget>
   <widget class="QLabel" name="img_frame">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>280</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">opacity: 10;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../Desktop/Projeto Desktop/Taskzen/image/itens_cadastro.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="Cadastrar">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>580</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat ExtraBold</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #E33C2A;
color: white;
border-radius: 5px;</string>
    </property>
    <property name="text">
     <string>Cadastrar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Voltar">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>580</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat ExtraBold</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #1F3043;
color: white;
border-radius: 5px;</string>
    </property>
    <property name="text">
     <string>Voltar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="user_input">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>230</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid #D2D2D2;
padding-left: 10px;
background: white;
border-radius: 5</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>exemplo123...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="email_input">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>320</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid #D2D2D2;
padding-left: 10px;
background: white;
border-radius: 5</string>
    </property>
    <property name="placeholderText">
     <string>exemplo@hotmail.com</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="senha_input">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>410</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid #D2D2D2;
padding-left: 10px;
background: white;
border-radius: 5;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="senha2_input">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>500</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1 solid #D2D2D2;
padding-left: 10px;
background: white;
border-radius: 5;</string>
    </property>
   </widget>
   <zorder>img_frame</zorder>
   <zorder>logo</zorder>
   <zorder>Taskzen</zorder>
   <zorder>Cadastro</zorder>
   <zorder>txt_usuario</zorder>
   <zorder>txt_email</zorder>
   <zorder>txt_password</zorder>
   <zorder>txt_password2</zorder>
   <zorder>Cadastrar</zorder>
   <zorder>Voltar</zorder>
   <zorder>user_input</zorder>
   <zorder>email_input</zorder>
   <zorder>senha_input</zorder>
   <zorder>senha2_input</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
